docs/library-io.md

```diff
@@ -173,7 +173,7 @@ That's it. The reason we don't have a `ParseJSON()` is that golang, like almost
   // generating a gff string that then can be piped to stdout or written to a database.
   ecoliGffString := BuildGff(ecoliAnnotatedSequence)
 
-  ecoliGbkString := BuildGbk(ecoliAnnotatedSequence)
+  ecoliGbkString := Build(ecoliAnnotatedSequence)
```

Comment:
There is a `BuildGff` right above this. Naming is inconsistent/confusing.

Maybe make a universal `Build(..., 'GBK')` interface?

Reply:
I was actually going to remove these docs for good with this PR. Too likely to break and not helpful. Should be replaced with new tutorials that are coming in new PR.

> Maybe make a universal Build(..., 'GBK') interface?

Possibly but I really like the current interface which is `genbank.Build(Sequence)`